plugins/ohifv3/extensions/monai-label/src/components/MonaiLabelPanel.tsx

```diff
@@ -21,146 +36,214 @@ export default class MonaiLabelPanel extends Component {
   };
 
   notification: any;
-  settings: any;
-  state: { info: {}; action: {} };
+  settings;
   actions: {
-    options: any;
     activelearning: any;
     segmentation: any;
-    smartedit: any;
+    pointprompts: any;
+    classprompts: any;
   };
-  props: any;
-  SeriesInstanceUID: any;
-  StudyInstanceUID: any;
+  serverURI = 'http://127.0.0.1:8000';
```

Comment:
should this be hardcoded?

Reply:
it's just the default value.  otherwise url is fetched from input